src/odemis/gui/comp/overlay/world.py

```diff
@@ -3072,6 +3072,7 @@ def draw(self, ctx, shift=(0, 0), scale=1.0):
 
             self.position_label.pos = pos
             self.position_label.text = size_lbl
+            self.position_label.colour = self.colour  # same color as ROA
```

Comment:
I see you removed this in the next commit. I'm also not sure this will actually improve readability. Bright green or yellow text does not always work well on a gray background either. Let's think about this a bit more before changing it and maybe talk to Maxim or someone with a bit of design background.

Reply:
No it is not removed but added. This is the commit after the other one. How about we choose the Odemis blue or orange? That should be readable. I can also check with Maxim what color he would suggest.

Reply:
The labels with couloured text always have a "shadow" below them to help readability, but arguably, it can be difficult still sometimes. Another approach could be to do as we do for the "Value under cursor", which draws a whole grey box below. I think that ideally we'd have some thin black borders around the letters, like subtitles in movies, but I don't know how to do that with Cairo....

Reply:
I like the grey box, that does make it more readable and sounds like it's not that hard to implement.

Reply:
I found a way to do it, but in the end it took me way longer then thought :laughing: I had to make a change in the Label base class. Please try it out to see if you like it like it is now. I also do not know how the changes in the base class are affecting any other systems. 
I also found a way to draw black borders around letters. I got it working by changing the base class as well. However, finding a good trade off between border and filling was difficult with the small labels. But maybe there are other ways as well. This is what I found - it needs to replace `ctx.show_text("text")`.
```
ctx.set_source_rgb(1, 1, 1)
ctx.text_path(part)
ctx.fill_preserve()
ctx.set_source_rgb(0, 0, 0)
ctx.set_line_width(0.5)
ctx.stroke()
```